spec/file_reference_installer/compile_source_file_reference_installer_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+#TODO: Figure out shorter class name
+module Xcake
+  describe CompileSourceFileReferenceInstaller do
+    before :each do
+      @node = double("Node")
+    end
+
+    it "should not able to install folder" do
+      allow(@node).to receive(:path).and_return('Folder/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.